Sources/MusicXML/Complex Types/Note.swift

```diff
@@ -17,6 +17,12 @@ public struct Note {
 
     // MARK: - Attributes
 
+    public var position: Position = Position()
+    public var fontFamily: CommaSeparatedText?
+    public var fontStyle: FontStyle?
+    public var fontSize: FontSize?
+    public var fontWeight: FontWeight?
```

Comment:
As I mentioned in the PR thread, can these be clumped together as `Font`?

Reply:
Yep